mediator/src/main/scala/io/iohk/atala/mediator/app/MediatorAgent.scala

```diff
@@ -327,16 +228,21 @@ object MediatorAgent {
           )
 
         } yield (ret)
-      case req @ Method.POST -> !! if req.headersAsList.exists { h =>
-            h.key.toString.toLowerCase == "content-type" &&
-            (h.value.toString.startsWith(MediaTypes.SIGNED.typ) ||
-              h.value.toString.startsWith(MediaTypes.ENCRYPTED.typ))
-          } =>
+      case req @ Method.POST -> Root
+          if req
+            .header(Header.ContentType)
```

Comment:
This method in `zio-http` is not working as expected. Maybe a Bug on version `3.0.0-RC2`

Reply:
I think it is related to this issue https://github.com/zio/zio-http/issues/2416